README.md

```diff
@@ -62,6 +62,18 @@ your component.
 - `options.content-exporter`: whether to configure sling model for content export
 - `options.model-adaptables`: array of adaptables to include in the Sling Model ('request' and/or 'resource')
 - `options.generic-javadoc`: whether to create generic javadoc for the getters in the model interface
+- `options.properties-tabs`: Properties to create in tabs. If empty, properties will be created without tab structure
```

Comment:
another nitpick - let's format this consistent with other documentation in this file.

"properties to create in tabs - if empty, properties will be created without tab structure"

Similar comment for `options.properties-shared-tabs` and `options.properties-global-tabs` below

Reply:
Updated the documentation, please let me know if anything is missed.